src/telemetry/SetTelemetryProps.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+export type SetTelemetryProps<Props extends object> = {
```

Comment:
Revisiting this, realized I'm overwriting a few telemetry properties that I hadn't intended, will update soon..